components/brave_talk/features.cc

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at https://mozilla.org/MPL/2.0/.
+
+#include "brave/components/brave_talk/features.h"
+#include "base/feature_list.h"
+
+namespace brave_talk {
+
+const base::Feature kBraveTalkFeature{"Brave Talk Integration",
```

Comment:
Consider this:
```
namespace features {

const base::Feature kBraveTalk{"BraveTalk", ...};

}  // namespace features
```
1. Dedicated namespace. This way you don't have to add ```Feature``` suffix to each feature.
2. No spaces in the feature name. You don't want spaces in command line or somewhere else where this feature can be used.
3. Feature name is mapped 1 to 1 to a variable name. ```kBraveTalk -> "BraveTalk"```. This just simplifies a lot of things.

Reply:
Makes a lot of sense 😄 Addressed in https://github.com/brave/brave-core/pull/9605/commits/d87ac4c24f34fa89ad43ae0259c4ae21b8caf2a3